sections/futures/Trade/TradePanelHeader.tsx

```diff
@@ -41,7 +39,7 @@ export default function TradePanelHeader({ accountType, onManageBalance }: Props
 		);
 	}
 
-	if (!balance && positionStatus.status === FetchStatus.Success) {
+	if (!!balance && balance.eq(0)) {
```

Comment:
I'd done some similar changes in the refactor PR which covers some extra cases for cross margin but yea this looks good and when I merge the refactor PR I'll make sure to bring back the case for cross margin.